verifyApiKey/data_helper_test.go

```diff
@@ -116,47 +116,3 @@ func setupApikeyCompanyTestDb(db apid.DB) {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(tx.Commit()).NotTo(HaveOccurred())
 }
-
-func setupKmsAttributesdata(db apid.DB) {
```

Comment:
Why delete these tests?

Reply:
I moved it to package `common`, `data_test.go`